deploy-manage/deploy/cloud-on-k8s.md

```diff
@@ -1,23 +1,98 @@
+
+- [](./cloud-on-k8s/deploy-an-orchestrator.md): ECK installation methods and configuration details.
+- [](./cloud-on-k8s/manage-deployments.md): Install and configure {{es}} clusters and {{kib}} instances through ECK.
+- [](./cloud-on-k8s/orchestrate-other-elastic-applications.md): Install and configure APM Server, Enterprise Search, Beats, Elastic Agent, Elastic Maps Server, and Logstash on Kubernetes.
+- [](./cloud-on-k8s/tools-apis.md): Collection of tools and APIs available in ECK based environments.
+
+## Looking for a quickstart? [eck-quickstart]
+
+If you want to get started quickly, follow these guides to deploy ECK and set up an {{es}} cluster:
+
+* [Install ECK using the YAML manifests](./cloud-on-k8s/install-using-yaml-manifest-quickstart.md)
+* [Deploy an {{es}} cluster](./cloud-on-k8s/elasticsearch-deployment-quickstart.md)
+* [Deploy a {{kib}} instance](./cloud-on-k8s/kibana-instance-quickstart.md)
+* [Update your deployment](./cloud-on-k8s/update-deployments.md)
+
+Afterwards, you can find further sample resources [in the project repository](https://github.com/elastic/cloud-on-k8s/tree/2.16/config/samples) or by checking out [our recipes](./cloud-on-k8s/recipes.md).
+
+## Supported versions [k8s-supported]
+
+ECK is compatible with:
+
+* Kubernetes 1.28-1.32
+* OpenShift 4.12-4.17
+* Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), and Amazon Elastic Kubernetes Service (EKS)
+* Helm: 3.2.0+
+* Elasticsearch, Kibana, APM Server: 6.8+, 7.1+, 8+
+* Enterprise Search: 7.7+, 8+
+* Beats: 7.0+, 8+
+* Elastic Agent: 7.10+ (standalone), 7.14+ (Fleet), 8+
+* Elastic Maps Server: 7.11+, 8+
+* Logstash: 8.7+
+
+ECK should work with all conformant installers as listed in these [FAQs](https://github.com/cncf/k8s-conformance/blob/master/faq.md#what-is-a-distribution-hosted-platform-and-an-installer). Distributions include source patches and so may not work as-is with ECK.
+
+Alpha, beta, and stable API versions follow the same [conventions used by Kubernetes](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning).
+
+Elastic Stack application images for the OpenShift-certified Elasticsearch (ECK) Operator are only available from version 7.10 and later.
+
+Check the full [Elastic support matrix](https://www.elastic.co/support/matrix#matrix_kubernetes) for more information.
+
+% TBD: discuss if these make sense here
+## Learn more about ECK [k8s_learn_more_about_eck]
+
+* [Orchestrate Elasticsearch on Kubernetes](https://www.elastic.co/elasticsearch-kubernetes)
+* [ECK post on the Elastic Blog](https://www.elastic.co/blog/introducing-elastic-cloud-on-kubernetes-the-elasticsearch-operator-and-beyond?elektra=products&storm=sub1)
+* [Getting Started With Elastic Cloud on Kubernetes (ECK)](https://www.youtube.com/watch?v=PIJmlYBIFXM)
+* [Running the Elastic Stack on Kubernetes with ECK](https://www.youtube.com/watch?v=Wf6E3vkvEFM)
```

Comment:
@shainaraskas : That section was already available in the original ECK overview page. Do you think we should keep it?